api-report/container-runtime.api.md

```diff
@@ -315,6 +317,12 @@ export interface IContainerRuntimeOptions {
     loadSequenceNumberVerification?: "close" | "log" | "bypass";
     // (undocumented)
     summaryOptions?: ISummaryRuntimeOptions;
+    useDataStoreAliasing?: boolean;
+}
+
+// @public
+export interface IDataStore extends IFluidRouter {
```

Comment:
It might be worth run a quick Q&A on public forum to see if IDataStore name resonates with people.